api/ws/workers/middlewares/handshake.js

```diff
@@ -157,7 +157,11 @@ var extractHeaders = function(request) {
 	headers.ip = request.remoteAddress.split(':').pop();
 	headers.httpPort = +headers.httpPort;
 	headers.wsPort = +headers.wsPort;
-	if (headers.height != null) {
+
+	if (
+		(headers.height && headers.height !== null) ||
```

Comment:
So if `headers.height` is true, it wasn't null or undefined so the following checks are redundant. 

Should be `headers.height !== undefined || headers.height !== null`